src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ScoreSystem.java

```diff
@@ -203,17 +190,10 @@ private void resetLevel() {
                 continue;
             }
 
-            movePlayerFlagToBase(playerWithFlag, oppositionTeam, heldFlag);
+            playerWithFlag.send(new MoveFlagToBaseEvent(heldFlag,oppositionTeam));
```

Comment:
Ah, we use the same event for "**return** flag to own base" and "**reset** flag on score"... In both cases, the flag is returned to the base of the team the flag represents, so I still think we don't need the target base as explicit parameter.